### ISSUE TITLE
migration with `-replicas` option > 1 creates as many replicas as there are pools

### ISSUE THREAD

**Issue by DmitryLitvintsev** [dCache/dcache#7509]
Demonstration:
```
# ssh -p 22224 admin@example.org "\s w-cmsstor708-disk-disk2 migration copy -tmode=cached -select=random -pnfsid=00007202AED0F7DC45F2B399CAE2297DCAF1 -replicas=2 -target=pgroup flushPools"
[5] INITIALIZING migration copy -tmode=cached -select=random -pnfsid=00007202AED0F7DC45F2B399CAE2297DCAF1 -replicas=2 -target=pgroup -- flushPools

[# ssh -p 22224 admin@example.org "\sl 00007202AED0F7DC45F2B399CAE2297DCAF1 rep ls 00007202AED0F7DC45F2B399CAE2297DCAF1"
w-cmsstor708-disk-disk2:
    00007202AED0F7DC45F2B399CAE2297DCAF1 <C-------X--L(0)[1]> 2490786864 si={cms.cms11}
w-cmsstor905-disk-disk3:
    00007202AED0F7DC45F2B399CAE2297DCAF1 <C----------L(0)[0]> 2490786864 si={cms.cms11}

# ssh -p 22224 admin@example.org "\sl 00007202AED0F7DC45F2B399CAE2297DCAF1 rep ls 00007202AED0F7DC45F2B399CAE2297DCAF1"
w-cmsstor201-disk-disk2:
    00007202AED0F7DC45F2B399CAE2297DCAF1 <C-------X--L(0)[0]> 2490786864 si={cms.cms11}
w-cmsstor708-disk-disk2:
    00007202AED0F7DC45F2B399CAE2297DCAF1 <C-------X--L(0)[1]> 2490786864 si={cms.cms11}
w-cmsstor905-disk-disk3:
    00007202AED0F7DC45F2B399CAE2297DCAF1 <C----------L(0)[0]> 2490786864 si={cms.cms11}
w-cmsstor920-disk-disk4:
    00007202AED0F7DC45F2B399CAE2297DCAF1 <C-------X--L(0)[0]> 2490786864 si={cms.cms11}

# ssh -p 22224 admin@example.org "\sl 00007202AED0F7DC45F2B399CAE2297DCAF1 rep ls 00007202AED0F7DC45F2B399CAE2297DCAF1"
w-cmsstor106-disk-disk3:
    00007202AED0F7DC45F2B399CAE2297DCAF1 <C-------X--L(0)[0]> 2490786864 si={cms.cms11}
w-cmsstor201-disk-disk2:
    00007202AED0F7DC45F2B399CAE2297DCAF1 <C-------X--L(0)[0]> 2490786864 si={cms.cms11}
w-cmsstor708-disk-disk2:
    00007202AED0F7DC45F2B399CAE2297DCAF1 <C-------X--L(0)[1]> 2490786864 si={cms.cms11}
w-cmsstor905-disk-disk3:
    00007202AED0F7DC45F2B399CAE2297DCAF1 <C----------L(0)[0]> 2490786864 si={cms.cms11}
w-cmsstor920-disk-disk4:
    00007202AED0F7DC45F2B399CAE2297DCAF1 <C-------X--L(0)[0]> 2490786864 si={cms.cms11}
```

**Comment by greenc-FNAL** [comment]
Fixed by https://github.com/dCache/dcache/commit/218dbe01f71450e4de6693c51951bbecbbd5bda3.

**Comment by greenc-FNAL** [comment]
Closed by 218dbe01f71450e4de6693c51951bbecbbd5bda3

**Comment by greenc-FNAL** [comment]
Fixed by https://github.com/dCache/dcache/commit/218dbe01f71450e4de6693c51951bbecbbd5bda3
